docs-ref-services/data-lake-analytics.md

```diff
@@ -12,41 +12,24 @@ ms.devlang: nodejs
 ms.service: Data Lake Analytics
 ---
 
-# Azure Data Lake Analytics modules for Node.js
+# Azure Data Lake Analytics modules for JavaScript
 
 Azure Data Lake Analytics is an on-demand analytics job service to simplify big data analytics. You can focus on writing, running, and managing jobs rather than on operating distributed infrastructure. Instead of deploying, configuring, and tuning hardware, you write queries to transform your data and extract valuable insights. The analytics service can handle jobs of any scale instantly by setting the dial for how much power you need. You only pay for your job when it is running, making it cost-effective. The analytics service supports Azure Active Directory letting you manage access and roles, integrated with your on-premises identity system. It also includes U-SQL, a language that unifies the benefits of SQL with the expressive power of user code. U-SQL’s scalable distributed runtime enables you to efficiently analyze data in the store and across SQL Servers in Azure, Azure SQL Database, and Azure SQL Data Warehouse.
 
 ### Management package
 
 ### Install the npm module
 
-Use npm to install the Azure Data Lake Analytics modules for Node.js
+Use npm to install the Azure Data Lake Analytics modules for JavaScript
 
 ```bash
 npm install azure-arm-datalake-analytics

```

Comment:
`@azure/arm-datalake-analytics` ?

Reply:
Couldn't find the newer ARM package for this one. Do you know the new package name for this service?

Reply:
Good point, I couldnt find it either. In this case, please revert all changes to this file